.yarnrc.yml

```diff
@@ -11,9 +13,3 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs
     spec: "https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js"
-  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
-    spec: "@yarnpkg/plugin-workspace-tools"
-  - path: .yarn/plugins/@yarnpkg/plugin-constraints.cjs
-    spec: "@yarnpkg/plugin-constraints"
-
-yarnPath: .yarn/releases/yarn-3.3.0.cjs
```

Comment:
It is no longer necessary to store the Yarn binary in the repo, as Yarn can rely on the version that Corepack installs: https://github.com/yarnpkg/berry/pull/4254